src/wing/CCPACSWingCell.cpp

```diff
@@ -189,7 +188,7 @@ bool CCPACSWingCell::IsInside(double eta, double xsi) const
     }
 }
 
-void CCPACSWingCell::ReadCPACS(TixiDocumentHandle tixiHandle, const std::string &cellXPath)
+void CCPACSWingCell::ReadCPACS(const TixiDocumentHandle& tixiHandle, const std::string &cellXPath)
```

Comment:
I the handle was just an integer, why don't you just use an integer than?
```
ReadCPACS(int handle, ....)
```
Probably because you wanted to abstract the true underlying representation of a "tixi handle" away from the user. The user does not need to know, what a TixiDocumentHandle really is.

If we don't know what a certain type exactly is, what do the following two interfaces tell us about a type T?
```
void func(T t);
void func(const T& t);
```
In the first version, an instance of T is copied when func() is called. Or in other words: func() gets a copy of a T of its own, each time it is called, owning that copy for the lifetime of the call to func() and destroying the copy afterwards.
In the second version, func() has read only access to an instance of T. No copy is performed, func() does not take ownership of anything, it just may read from an outside instance of T.
In my opinion, the second version matches our situation better. ReadCPACS() show be provided unmodifyable access to the resource abstracted from a TixiDocumentHandle.

A further thought: If we ever want to change the definition of TixiDocumentHandle, e.g. make it a class (similar to TixiDocument in the generator) which holds ownership of a document (e.g. closing it in dtor), we must use the reference version. Thus, the reference version is also forward compatible.

Apart from these reasons, I really don't care (for now) if you want to pass by value or by reference. I prefere the reference version, but I can change it back. What do you think?